### PR TITLE
Use the new webhook image in deployment/CI/CSV

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ spec:
         - name: APP
```
Comment (zcahana) [inline deploy/operator.yaml:112]: Is the `WEBHOOK_MODE` env var still needed?

**Review by zcahana — COMMENTED** [review]
Just a tiny comment, otherwise lgtm.

**Comment by nunnatsa** [comment]
> Just a tiny comment, otherwise lgtm.

Yes, because of the upgrade tests, that uses the same csv, but witb previous image.

**Comment by zcahana** [comment]
> Yes, because of the upgrade tests, that uses the same csv, but witb previous image.

ok then,
/lgtm

**Pull request merged**